spec/requests/submissions_spec.rb

```diff
@@ -111,4 +111,129 @@
+          expect(response.body).to include(not_started_submission.id.to_s)
+          expect(response.body).to include(eligible_submission.id.to_s)
+          expect(response.body).not_to include(selected_submission.id.to_s)
+          expect(response.body).not_to include(in_progress_submission.id.to_s)
+          expect(response.body).not_to include(completed_submission.id.to_s)
+        end
+
+        it 'shows only completed submissions' do
+          get submissions_phase_path(phase), params: { status: 'completed' }
+
+          expect(response.body).to include(completed_submission.id.to_s)
+          expect(response.body).to include(selected_submission.id.to_s)
+          expect(response.body).not_to include(not_started_submission.id.to_s)
+          expect(response.body).not_to include(in_progress_submission.id.to_s)
+          expect(response.body).not_to include(eligible_submission.id.to_s)
+        end
+      end
+
+      context 'when filtering by eligibility' do
+        it 'displays only eligible for evaluation submissions' do
+          get submissions_phase_path(phase), params: { eligible_for_evaluation: 'true' }
+
+          expect(response.body).to include(eligible_submission.id.to_s)
+          expect(response.body).to include(selected_submission.id.to_s)
+          expect(response.body).not_to include(not_started_submission.id.to_s)
+          expect(response.body).not_to include(in_progress_submission.id.to_s)
+          expect(response.body).not_to include(completed_submission.id.to_s)
+        end
+
+        it 'displays only selected to advance submissions' do
+          get submissions_phase_path(phase), params: { selected_to_advance: 'true' }
+
+          expect(response.body).to include(selected_submission.id.to_s)
+          expect(response.body).not_to include(eligible_submission.id.to_s)
+          expect(response.body).not_to include(not_started_submission.id.to_s)
+          expect(response.body).not_to include(in_progress_submission.id.to_s)
+          expect(response.body).not_to include(completed_submission.id.to_s)
+        end
+      end
+
+      context 'when sorting submissions' do
+        before do
+          create(:evaluation,
+            evaluator_submission_assignment: create(:evaluator_submission_assignment, submission: in_progress_submission),
+            total_score: 80
+          )
+
+          create(:evaluation,
+            evaluator_submission_assignment: create(:evaluator_submission_assignment, submission: completed_submission),
+            total_score: 90
+          )
+        end
+
+        it 'orders submissions by score high to low' do
+          get submissions_phase_path(phase), params: { sort: 'average_score_high_to_low' }
+
+          response_body = response.body
+          high_score_index = response_body.index(completed_submission.id.to_s)
+          low_score_index = response_body.index(in_progress_submission.id.to_s)
+
+          expect(high_score_index).to be < low_score_index
```

Comment:
this spec is also failing I believe because of the same broadness in finding content on the page. we have [Capybara::RSpecMatchers](https://rubydoc.info/github/jnicklas/capybara/Capybara/RSpecMatchers) available for use in request specs. take a look at those and see if something might be more helpful.